### ISSUE TITLE
mac address on door sensors 0/0/0/0

### ISSUE THREAD

**Issue by teslawave** [kevinvincent/ha-wyzesense#70]
having an issue with this add-on first the entities would show up then show as unavailable but still worked in all automation, then they all disappeared and when i went to add them only the motion sensor would give mac address, the two door sensors give me a mac of 0 so i can't add or remove them with the service call.  So i can only ever add one now as when i add the other it just shows up as the same mac address and over rides the previous one.

mac: "\0\0\0\0\0\0\0\0"
device_class: door
timestamp: '2019-12-02T22:23:24.030000'
rssi: -67
battery_level: 92
friendly_name: front door

core-ssh:~# dmesg | grep hidraw
[    0.791484] hidraw: raw HID events driver (C) Jiri Kosina
[    2.632003] hid-generic 0003:1A86:E024.0001: hiddev96,hidraw0: USB HID v1.00 Device [HID 1a86:e024] on usb-3f980000.usb-1.3/input0
finally my log 
[Log](https://pastebin.com/wK2iERK6)

**Comment by teslawave** [comment]
is there anyway to edit the wyze config file in order to enter an id in?

**Comment by JFC-Dev** [comment]
Do you have a fast blink light every ~10 seconds on the sensors that have ni mac address? I read in a forum that the battery might be dying when it happens. I had this issue but the battery was new, removing the battery multiple times didn't help. In the end I fixed the issue by removing the battery, pushing the pin that connects to the battery away from the board and then putting the battery back in. So far, no more blinking light every 10 seconds and I the sensor can be added with its MAC address now!

**Comment by smoke007** [comment]
@teslawave  Yes, see my post here https://github.com/kevinvincent/ha-wyzesense/issues/130#issuecomment-624960162
